compiler/IREmitter/IREmitter.cc

```diff
@@ -866,7 +866,7 @@ void IREmitter::buildInitFor(CompilerState &cs, const core::SymbolRef &sym, stri
     builder.SetInsertPoint(bb);
 
     if (IREmitterHelpers::isFileOrClassStaticInit(cs, sym)) {
-        // We include sorbet_version.c when compiling sorbet_llvm itself to get the expected version.
+        // We include sorbet_version.c when compiling the Sorbet Compiler itself to get the expected version.
```

Comment:
Just out of curiosity, did we verify that this still works?

Reply:
I did not, but I see no reason why it would have stopped working. The way we compute the version information is using git information from the repo where Bazel is, which in the case of this line is the Git SHA of Stripe's codebase at the time that the Sorbet version is bumped.